packages/shared/src/components/sidebar/SidebarBottomSection.tsx

```diff
@@ -54,15 +51,6 @@ export function SidebarBottomSectionSection({
     },
   ];
 
-  if (showHiring) {
```

Comment:
Why do we move it from here?

Reply:
It was asked to be removed completely.

Relevant discussion: https://dailydotdev.slack.com/archives/C02JAUF8HJL/p1693840704759149?thread_ts=1693840563.128999&cid=C02JAUF8HJL